csrc/pfcustom.c

```diff
@@ -63,12 +63,18 @@ static void CTest1( cell_t Val1, cell_t Val2 )
 ** It is called by the pForth kernel.
 */
 #define NUM_CUSTOM_FUNCTIONS  (2)
-CFunc0 CustomFunctionTable[NUM_CUSTOM_FUNCTIONS];
+CFunc0 CustomFunctionTable[NUM_CUSTOM_FUNCTIONS + PF_WORDLIST_EXPORT_FUNCTIONS];
 
 Err LoadCustomFunctionTable( void )
 {
 	CustomFunctionTable[0] = CTest0;
 	CustomFunctionTable[1] = CTest1;
+        /* Insert your functions here. */
+        
+#ifdef PF_SUPPORT_WORDLIST
+        CustomFunctionTable[NUM_CUSTOM_FUNCTIONS + 0] = ffInitWordLists;

```

Comment:
The custom functions are for others to extend pForth. These words would be part of pForth and should have IDs like the other kernel words.